package.json

```diff
@@ -6,15 +6,16 @@
   "main": "./js/index.js",
   "types": "./js/index.d.ts",
   "scripts": {
-    "preversion": "npm run compile && npm test",
+    "preversion": "npm run generate-barrels && npm run compile && npm test",
     "precompile": "npm run clean",
     "compile": "tsc",
     "test": "set NODE_ENV=test && mocha -t 6000 -R spec js/tests --grep @external --invert",
     "test-all": "set NODE_ENV=test && mocha -t 6000 -R spec js/tests",
     "coverage": "istanbul cover node_modules/mocha/bin/_mocha -- -t 6000 -R spec js/tests",
-    "clean": "del-cli js/",
+    "clean": "rm -rf js/",
```

Comment:
Doesn't this prevent usage in windows environments?

Reply:
Prevents developing in windows environment.